### ISSUE TITLE
AttributeError: type object 'Variable' has no attribute 'chunk'

### ISSUE THREAD

**Issue by MrLinNing** [tbung/revnet#7]
hello, @tbung 
My Pytorch version is 0.4.0, cuda version is 9.0 and python is 3.5.
But when I run this command:
```
python train_cifar.py --model revnet38 --clip 0.25
```
Problem happened:
```

File "pytorch-revnet/revnet/revnet.py", line 191, in _grad 
        dy1, dy2 = Variable.chunk(dy, 2, dim=1) 
AttributeError: type object 'Variable' has no attribute 'chunk'
```
Can you help me to fix the bug? Thank you.

**Comment by tbung** [comment]
Hi @MrLinNing,

this bug is due to the fact that this code was written when pytorch 0.4 was not released yet, and a bunch of stuff changed since then, as described in the [migration guide](https://pytorch.org/blog/pytorch-0_4_0-migration-guide/).

I will try and fix all related issues, but I currently have other stuff I need to work on, so it might take some time.

You can of course try and fix all related issues by yourself, your bug should be resolved by replacing `Variable.chunk` with `torch.chunk`. Pull requests are very welcome if you do!

Thanks for your interest and patience!

Cheers,
Till

**Comment by tbung** [comment]
This has been closed by @JuanFMontesinos in #8.
